augur/export_v2.py

```diff
@@ -838,8 +839,18 @@ def run_v2(args):
     # Write outputs - the (unified) dataset JSON intended for auspice & perhaps the ref root-sequence JSON
     indent = {"indent": None} if args.minify_json else {}
     write_json(data=data_json, file_name=args.output, include_version=False, **indent)
-    if 'reference' in node_data:
-        write_json(data=node_data['reference'], file_name=args.output[:-5]+'_root-sequence.json', include_version=False, **indent)
+
+    if args.include_root_sequence:
+        if 'reference' in node_data:
+            # Save the root sequence with the same stem from the main auspice
+            # output filename.  For example, if the main auspice output is
+            # "auspice/zika.json", the root sequence will be
+            # "auspice/zika_root-sequence.json".
+            output_path = Path(args.output)
+            root_sequence_path = output_path.parent / Path(output_path.stem + "_root-sequence" + output_path.suffix)
+            write_json(data=node_data['reference'], file_name=root_sequence_path, include_version=False, **indent)
+        else:
+            warning("Root sequence output was requested, but the node data provided is missing a 'reference' key.")
```

Comment:
This feels like it should be a fatal error to me instead of a warning: Augur was asked to do a thing and cannot do the thing. I'd want to know loud and clear that it couldn't, which would leave me with the choice of either removing the flag from my Augur invocation or fixing the node data input.